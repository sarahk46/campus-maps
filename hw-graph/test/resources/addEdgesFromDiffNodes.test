# Tests that a graph can be created and 5 nodes
# and 4 edges can be made from diff nodes.

# Create a graph
CreateGraph graph1

# Add 5 nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5

#Add 4 edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n3 e2
AddEdge graph1 n3 n4 e3
AddEdge graph1 n4 n5 e4