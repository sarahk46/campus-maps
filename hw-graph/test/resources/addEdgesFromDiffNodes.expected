# Tests that a graph can be created and 5 nodes
# and 4 edges can be made from diff nodes.

# Create a graph
created graph graph1

# Add 5 nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1

#Add 4 edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n3 in graph1
added edge e3 from n3 to n4 in graph1
added edge e4 from n4 to n5 in graph1